var/spack/repos/builtin/packages/gmsh/package.py

```diff
@@ -30,6 +35,8 @@ class Gmsh(CMakePackage):
     version('2.12.0', sha256='7fbd2ec8071e79725266e72744d21e902d4fe6fa9e7c52340ad5f4be5c159d09')
     version('develop', branch='master', git='https://gitlab.onelab.info/gmsh/gmsh.git')
 
+    patch('gmsh-4.7.1-cgns-type.patch', sha256='bee4401602d114cbd011336261cd90a69d8cd80f4472f59ce9dfc65e8850dddb', when='@4.7.1')
```

Comment:
If you do:
```python
patch('https://gitlab.onelab.info/gmsh/gmsh/-/commit/b0b3ece698718bf1a79f50778d24d51c207f8727.patch',
      sha256='...')
```
then you don't need to store such a large patch in Spack.

Reply:
Thanks for that advice. But the patch commit cannot be applied to v4.7.1 directly. 
Any better suggestions? Or I can just delete this patch wait for the new versions to support CGNS.

Reply:
Hmm, in that case maybe just delete the patch and wait for a new release...